security/.htaccess

```diff
@@ -13,18 +13,24 @@ RewriteRule ^v3-unstable$ https://w3c-ccg.github.io/security-vocab/contexts/secu
 # https://w3id.org/security/data-integrity
 
 RewriteRule ^data-integrity$ https://www.w3.org/TR/vc-data-integrity/ [R=302,L]
-RewriteRule ^data-integrity/v1$ https://www.w3.org/ns/data-integrity/v1 [R=302,L]
+RewriteRule ^data-integrity/v1$ https://w3c.github.io/vc-data-integrity/contexts/data-integrity/v1 [R=302,L]
+RewriteRule ^data-integrity/v2$ https://w3c.github.io/vc-data-integrity/contexts/data-integrity/v2 [R=302,L]
 
 # https://w3id.org/security/multikey
 
-RewriteRule ^multikey$ https://w3c.github.io/vc-data-integrity/ [R=302,L]
-RewriteRule ^multikey/v1$ https://digitalbazaar.github.io/multikey-context/contexts/multikey-v1.jsonld [R=302,L]
+RewriteRule ^multikey$ https:/w3id.org/security/ [R=302,L]
+RewriteRule ^multikey/v1$ https://w3c.github.io/vc-data-integrity/contexts/multikey/v1 [R=302,L]
+
+# https://w3id.org/security/jwk
+
+RewriteRule ^jwk$ https:/w3id.org/security/ [R=302,L]
+RewriteRule ^jwk/v1$ https://w3c.github.io/vc-data-integrity/contexts/jwk/v1 [R=302,L]
 
 # https://w3id.org/security/suites
 
 # TODO: update registry to reflect this file.
 
-RewriteRule ^suites$ https://w3c-ccg.github.io/ld-cryptosuite-registry [R=302,L]
+RewriteRule ^suites$ https://w3c.github.io/vc-specs-dir/ [R=302,L]
```

Comment:
Is the theory more suites will be listed at this page?  It doesn't list all the things in the previous ld-cryptosuite-registry, or all the suites below.

Reply:
Yes, it will list more suites in time. The old ld-cryptosuite-registry is abandoned at this point, long live vc-specs-dir!